01.fit_a_line/README.cn.md

```diff
@@ -173,112 +155,137 @@ use_cuda = False
 place = fluid.CUDAPlace(0) if use_cuda else fluid.CPUPlace()
 ```
 
-### 创建训练器
-训练器会读入一个训练程序和一些必要的其他参数：
+除此之外，还可以定义一个事件响应器来处理类似`打印训练进程`的事件：
 
 ```python
-trainer = Trainer(
-    train_func=train_program,
-    place=place,
-    optimizer_func=optimizer_program)
+# Plot data
+from paddle.v2.plot import Ploter
+
+train_title = "Train cost"
+test_title = "Test cost"
+plot_cost = Ploter(train_title, test_title)
+
+
+def event_handler(title, loop_step, handler_val):
```

Comment:
Event Handler is not needed when using low-level APIs.

You can just plot/print in-placely.

Reply:
ok, thanks review!

Reply:
Has been updated at 3d0283e